jujugui/static/gui/src/app/utils/test-statsd.js

```diff
@@ -71,11 +71,24 @@ describe('StatsClient', () => {
     assertXHRSent(url, 'gui.awesome:1|c');
   });
 
-  it('can add flags as statsd tags', () => {
+  it('adds flags as statsd tags', () => {
     const url = 'https://example.com/stats/';
     const flags = {'test.bar': true, 'baz': true};
     const client = new window.jujugui.StatsClient(url, 'gui', flags);
     const name = client._addFlags('foo');
     assert.equal(name, 'foo,test.bar=true');
   });
+
+  it('adds multiple flags as statsd tags', () => {
+    const url = 'https://example.com/stats/';
+    const flags = {
+      'test.bar': true,
+      baz: true,
+      'test.bad': false,
```

Comment:
Yes you are right, but given we established the contract that flags are a map, and given  that checking the value truthiness is very easy, I added the check so that we don't rely on a convention established by the client.